src/v2/Components/Toast/ToastComponent.tsx

```diff
@@ -53,6 +53,7 @@ const ToastComponent: React.FC<ToastComponentProps> = ({
         boxShadow:
           "0px 0px 1px 2rgba (0, 0, 0, 0.08), 0px 2px 8px rgba(0, 0, 0, 0.12)",
         left: isMoveToRight ? "100%" : 0,
+        zIndex: 1,
```

Comment:
I think that the `Box` component takes a `zIndex` prop directly, which we prefer doing over adding to the style prop. So, adding a `zIndex={1}` directly should do the trick.

Reply:
Updated to zIndex as a prop.